packages/generator/pyproject.toml

```diff
@@ -75,3 +75,6 @@ skips = [
   "B101", # assert_used
   "B702", # use_of_mako_templates
 ]
+
+[tool.ni-python-styleguide]
```

Comment:
Keep tool.black and tool.ni-python-styleguide together and keep their excludes in sync. Note that black uses regex instead of comma-separated list.

Example: https://github.com/ni/measurement-plugin-python/blob/main/packages/service/pyproject.toml#L1